gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -601,26 +633,41 @@ def assign_pops(
             pop_pca_scores_ht,
             pc_cols=[pop_pca_scores_ht.scores[i - 1] for i in pcs],
             known_col="training_pop",
-            min_prob=min_prob,
             output_col=pop_field,
+            min_prob=min_prob,
+            missing_label=missing_label,
         )
 
         pop_ht = pop_ht.annotate(
```

Comment:
I suppose a lot of this is very similar to above, so you could pull it out into a function, but not necessary